examples/zkapps/10-account-updates/src/ProofsOnlyZkApp.ts

```diff
@@ -17,7 +17,7 @@ export class ProofsOnlyZkApp extends SmartContract {
 
   deploy(args: DeployArgs) {
     super.deploy(args);
-    this.setPermissions({
+    this.account.permissions.set({
```

Comment:
we'll need doc updates for these changes, too.  How do we want to ensure that code changes in examples are reflected in the docs? (Separate PR? so we need an issue for each tutorial? ) 
For example, this code is shown in https://docs.minaprotocol.com/zkapps/tutorials/account-updates#non-upgradeable-proof-only

Reply:
Yes, I think at the moment we can just create PR's whenever there are code changes that also affect docs.

Reply:
@LucidSamuel @barriebyron Not sure, should I create separate PR for content update, or add it in this PR?

Reply:
I think you can simply add it to this one since we're keeping track of it.

Reply:
@LucidSamuel done